File-Manifests/NeMO-Audit-Manifests/README.md

```diff
@@ -0,0 +1,11 @@
+# NeMO Audit File Manifests
```

Comment:
Where will you leave instructions for how data submitters should interact with these files?  Do you want to create an additional set of instructions in here, or just do it by email?